mmtrack/models/trackers/quasi_dense_embed_tracker.py

```diff
@@ -0,0 +1,263 @@
+# Copyright (c) OpenMMLab. All rights reserved.
```

Comment:
The codes in this file are hard to understand.
Suggest to add many comments to explain the motivation.